### ISSUE TITLE
Float - over broken for L3 VNC stack

### ISSUE THREAD

**Issue by dosumis** [VirtualFlyBrain/VFB2#168]
Float-over => null in top left for most or all of VNC. In contrast, float-over works for brain.

This is likely a downstream effect of removing individuals for longitudinal subdivisions of VNC.  Do we need to remove corresponding indexes on stack?

**Comment by Robbie1977** [comment]
yep we need to provide a reduced set of id's for the most specific neuropil regions. @dosumis Do we have a way to do this with the current neo4j labels or does this require more work?

**Comment by dosumis** [comment]
You should be able to pull back stack index numbers from the IRW edge.  On PDB that will => only image in the release. ie. it will exclude indexes for longitudinal subdivisions of VNC.  Is that sufficient?
Curious about how this part of the pipeline works.  Seems like it could/should be hardened/automated.
